### DISCUSSION TITLE
Custom pgpass file

### DISCUSSION THREAD

**Discussion by anthonator** [cloudnative-pg/cloudnative-pg#2819] [Q&A]
We need a custom pgpass file that has credentials for multiple clusters. Is there a way to do this with CloudNativePG? We tried creating the pgpass file and mounting it as a projected volume but CloudNativePG prevents us from adding a`PGPASSFILE` environment variable. There doesn't seem to be any mention of pgpass files in the documentation other than what's provided by the secrets.

**Comment by gbartolini** [comment]
Is this because you are trying to use foreign data wrappers or logical subscriptions?

**Reply by anthonator** [reply]
This is specifically for a multi-node Citis cluster
